src/index.js

```diff
@@ -59,33 +42,20 @@ app.use((req, res, next) => {
 if (!isProduction) {
   // eslint-disable-next-line no-unused-vars
   app.use((err, req, res, next) => {
-    serverLog(err.stack);
-
     res.status(err.status || 500);
-
     res.json({
-      errors: {
-        message: err.message,
-        error: err,
-      },
+      success: false,
```

Comment:
You are nt providing a status code in your response body.

Reply:
During code review today, TTL asked that it should be changed. It is already in the response status

Reply:
During code review today, TTL asked that it should be changed. It is already in the response status

Reply:
You mean it should just be true or value